.github/actions/fastlane/action.yaml

```diff
@@ -0,0 +1,58 @@
+name: 'fastlane'
+description: 'Runs fastlane with target'
+inputs:
+  environment:
+    required: true
+  platform:
+    reauired: true
+  target:
+    required: true
+  P12:
+    description: "P12 cert for ios sign"
+    required: false
+    default: ""
+  RELEASE_KEYSTORE:
+    description: "RELEASE_KEYSTORE for android sign"
+    required: false
+    default: ""
+  RELEASE_KEYSTORE_PASS: 
+    description: "Password for android RELEASE_KEYSTORE"
+    required: false
+    default: ""
+runs:
+  using: "composite"
+  steps:
+    - name: Set up ruby
+      if: ${{ runner.name == 'Hosted Agent' }}
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '3.0'
+        bundler-cache: true
+    - name: Setup java
+      if: ${{ inputs.platform == 'android' }}
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'adopt'
+        java-version: '11'
```

Comment:
setup version as input param with default value, so you can easily "change" in future ci implementation reusing the workflow

Reply:
done